DigitalLearningSolutions.Web/Helpers/GenericSearchHelper.cs

```diff
@@ -73,7 +73,7 @@ public class GenericSearchHelper
             var results = Process.ExtractAll(
                 (T)query,
                 items,
-                item => item.SearchableName.ToLower(),
+                item => string.Join(" ", item.SearchableContent.Where(s => s != null)).ToLower(),

```

Comment:
Can this produce a situation where the search 'finds' a string that's actually composited from multiple separate content elements? I don't think it can because I think the search is on single words only not phrases-including-spaces, but I have no experience with the search system.

Reply:
Exactly how search behaves depends on the "scorer" used. 
In this context partial matches can be found as that is how it is configured.
If you look back at the tests in `AllDelegatesSearchTests`. The test cases, although basic, pass in multiple values as a search string and will find a match based on partial matches across multiple properties of the model.
This is expected and wanted behaviour.
So for example we have:
```
var delegate = new DelegateUser 
{
FirstName = "Emily",
LastName = "Harper",
CandidateNumber = "EH14"
};
```

We pass in the search string of `Haper H14` and it will find this delegate.